pkgs/by-name/mp/mpb/package.nix

```
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoreconfHook,
+  gfortran,
+  blas,
+  lapack,
+  fftw,
+  hdf5,
+  libctl,
+  guile,
+}:
+
+assert !blas.isILP64;
+assert !lapack.isILP64;
+
+stdenv.mkDerivation rec {
+  pname = "mpb";
+  version = "1.11.1";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+2cMjZSGdfngtGoAeZRPRPBDvflTEIOWO8Se0W6jv9k=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    gfortran
+  ];
+
+  buildInputs = [
+    blas
+    lapack
+    fftw
+    hdf5
+    libctl
+    guile
+  ];
+
+  enableParallelBuilding = true;
+
+  configureFlags = [
+    "--with-libctl=no"
+    "--enable-shared"
+    "--enable-maintainer-mode"
+    "--disable-dependency-tracking"
+  ];
+
+  doCheck = false;
```

```suggestion
  doCheck = true;
```
With enabled libctl, guile and perl all tests pass. If possible, having tests is a good idea.